core/tests/fireproof/attachable-subscription.test.ts

```
@@ -0,0 +1,500 @@
+ * This would make React components update automatically when remote data arrives.
+ *
+ * FIX NEEDED:
+ * Core subscription system needs to treat remote data ingestion the same as local writes.
+ * Likely fix location: CRDT/ledger layer where remote data is applied to local database.
+ */
+
+describe("Remote Sync Subscription Tests", () => {
+  const sthis = ensureSuperThis();
+
+  // Subscription tracking variables
+  let subscriptionCallbacks: Array<() => void> = [];
```

ESLint

[error] Array type using 'Array<T>' is forbidden. Use 'T[]' instead.

(@typescript-eslint/array-type)